posthog/hogql_queries/insights/trends/trends_query_builder.py

```diff
@@ -706,7 +706,10 @@ def _events_filter(
                         elif property_clone["type"] == "person":
                             property_clone["key"] = f"events.person.properties.{property_clone['key']}"
                         property_clone["type"] = "data_warehouse"
-                    filters.append(property_to_expr(property_clone, self.team))
+                    expr = property_to_expr(property_clone, self.team)
+                    if property_clone["type"] in ("group", "element") and hasattr(expr, "left"):
+                        expr.left.chain = ["events", *expr.left.chain]
```

Comment:
Do we wanna check that this is indeed the correct `ast` node and not just check whether it has a `left` attribute? I presume you're testing that `expr` is a `ast.CompareOperation` and that `expr.left` is a `ast.Field` node?